SportsHub/SportsHub.DAL/Seeding/Seeder.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Newtonsoft.Json;
+using SportsHub.DAL.Data;
+using SportsHub.Domain.Models;
+
+namespace SportsHub.DAL.Seeding
+{
+    public class Seeder : ISeeder
+    {
+        #region Raw Data
+        private readonly string _roles = "[{\"Name\":\"User\"},{\"Name\":\"Admin\"}]";
```

Comment:
Why do you use json? Are you sure that it has correct format? Did you consider creating data generation method?